client/README.md

```diff
@@ -19,6 +19,7 @@ You will need the following installed in order to use the extension:
 - [Node v8.x or greater and npm v5.x or greater](https://nodejs.org/en/download/)
 - [Docker version v17.06.2-ce or greater](https://www.docker.com/get-docker)
 - [Docker Compose v1.14.0 or greater](https://docs.docker.com/compose/install/)
+- [Go version v1.12 or greater](https://golang.org/dl/)
```

Comment:
Should this be described as optional ? needed only for developing Go contracts ?